doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
  */
 ////
 
-calcurse(1)
+CALCURSE(1)
 ===========
 
-Name
+NAME
 ----
 
-calcurse - text-based organizer
+calcurse - terminal-based organizer for interactive and command line use
 
-Synopsis
+SYNOPSIS
 --------
 
-[verse]
-'calcurse' *-Q* [options] [*--from* <date>] [*--to* <date>|*--days* <num>]
-'calcurse' *-G* [options]
-'calcurse' *-i*<file>
-'calcurse' *-x*<format>
-'calcurse' *--gc*
-'calcurse' *--status*
-'calcurse' *--version*
-'calcurse' *--help*
-
-Description
------------
+--
+*calcurse* [-D 'directory'] | [-C 'confdir'] | [-c 'calendar-file']
```

Comment:
`datadir` instead of `confdir`?

Reply:
?? I considered `datadir` instead of `directory`.

Reply:
Yes, that's what I meant, sorry.

Reply:
I retained _directory_, because it contains both data and configuration files when `-D` is used alone.